packages/runtime/container-runtime/src/opLifecycle/batchManager.ts

```diff
@@ -55,13 +34,11 @@ export class BatchManager {
         // Cases where the message is still too large will be handled by the maxConsecutiveReconnects path.
         if (this.options.softLimit !== undefined
             && this.length > 0
-            && socketMessageSize >= this.options.softLimit
-            && Infinity === (this.options.compressionOptions?.minimumBatchSizeInBytes ?? Infinity)) {
```

Comment:
I can't figure out what this line was for. I think it said - if compression was disabled. But how is this different from limit of 1Tb that we never would hit?

Reply:
This line was disabling the size check if compression is enabled. So, if we're doing compression, we will always accept messages into the batch manager. This change kind of moves this check upwards (to the outbox, by configuring the batch manager to accept anything, based on its input limits) and dumbs down the batch manager.